packages/api-kit/src/SafeApiKit.ts

```diff
@@ -115,20 +115,28 @@ class SafeApiKit {
   /**
    * Decodes the specified Safe transaction data.
    *
-   * @param data - The Safe transaction data
+   * @param data - The Safe transaction data. '0x' prefixed hexadecimal string.
+   * @param to - The address of the receiving contract. If provided, the decoded data will be more accurate, as in case of an ABI collision the Safe Transaction Service would know which ABI to use
    * @returns The transaction data decoded
    * @throws "Invalid data"
    * @throws "Not Found"
    * @throws "Ensure this field has at least 1 hexadecimal chars (not counting 0x)."
    */
-  async decodeData(data: string): Promise<any> {
+  async decodeData(data: string, to?: string): Promise<any> {
```

Comment:
I don't see the correct type on the service:

https://safe-transaction-sepolia.staging.5afe.dev/#/data-decoder/data_decoder_create
https://github.com/safe-global/safe-transaction-service/blob/f446d55cb527358e5312cea5aa737e0bae1adc6c/safe_transaction_service/history/views.py#L1188-L1225

The return type i'm seeing does not match the { data, to } structure that is in reality the parameters. So not sure if that type is always going to be the one or it can change